br/pkg/lightning/config/config.go

```diff
@@ -35,12 +35,13 @@ import (
 	"github.com/pingcap/errors"
 	filter "github.com/pingcap/tidb-tools/pkg/table-filter"
 	router "github.com/pingcap/tidb-tools/pkg/table-router"
+	"go.uber.org/atomic"
+	"go.uber.org/zap"
```

Comment:
Nit: imports don't need to sort into three groups in TiDB repo. ditto for rest files.

Reply:
goland cann't set per project `imports format` config, will leave it